js/animateHighlight.js

```diff
@@ -0,0 +1,36 @@
+/*jshint unused: false */
+
+function addSpan( id ) {
+	var elm = $( '#' + id ),
+		string = elm.text(),
+		numCharacters = string.length,
+		newHTML = '',
+		i;
+
+	for ( i = 0; i < numCharacters; i++ ) {
+		if ( i === 0 ) {
+			newHTML = newHTML + '<span style="padding:1px 0px 1px 5px">' + string[ i ] + '</span>';
+		} else {
+			if ( i + 1 === numCharacters ) {
+				newHTML = newHTML + '<span style="padding:1px 5px 1px 0px">' + string[ i ] + '</span>';
+			} else {
+				newHTML = newHTML + '<span style="padding:1px 0px 1px 0px">' + string[ i ] + '</span>';
+			}
+		}
+	}
+	elm.html( newHTML );
+}
+
+function highlightSpan( id, num, color ) {
+	$( '#' + id + ' span:nth-child(' + num + ')' ).css( 'background-color', color );
```

Comment:
Why not setting a class instead of setting a color? could be easier to customize via CSS.